src/Utils/SchemaPrinter.php

```diff
@@ -211,7 +218,7 @@ protected static function printDescription(array $options, $def, $indentation =
     }
 
     /**
-     * @return string[]
+     * @return array<int, string>
```

Comment:
we can use list<> eventually

Reply:
Still not in PHPStan, but it would be nice.